AdyenCard/Utilities/CardPublicKeyProvider/CardPublicKeyProvider.swift

```diff
@@ -19,24 +19,29 @@ internal protocol AnyCardPublicKeyProvider: APIContextAware {
 
 /// :nodoc:
 internal final class CardPublicKeyProvider: AnyCardPublicKeyProvider {
-    
+
+    /// :nodoc:
+    private let request: ClientKeyRequest
+
     /// :nodoc:
-    internal let apiContext: APIContext
+    private let retryApiClient: AnyRetryAPIClient
     
     /// :nodoc:
     internal static var cachedCardPublicKey: String?
     
     /// :nodoc:
-    internal init(apiContext: APIContext,
-                  apiClient: AnyRetryAPIClient? = nil) {
-        self.apiContext = apiContext
-        if let apiClient = apiClient {
-            self.retryApiClient = apiClient
-        } else {
-            let scheduler = SimpleScheduler(maximumCount: 2)
-            self.retryApiClient = APIClient(apiContext: apiContext)
-                .retryAPIClient(with: scheduler)
-        }
+    internal convenience init(apiContext: APIContext) {
+        let scheduler = SimpleScheduler(maximumCount: 2)
+        self.init(apiClient: APIClient(apiContext: apiContext)
+            .retryAPIClient(with: scheduler),
+            request: ClientKeyRequest(clientKey: apiContext.clientKey))
```

Comment:
i think the indentation formatting on these lines is a bit off

Reply:
This is because of trailing `.retryAPIClient(with: scheduler)`
👍